### PR TITLE
Creating the about page

### PR DESCRIPTION
**Pull request by TylerCarey98**
-Added About information to the about page
-Made it look pretty

### CODE REVIEW

File: src/DnDTracker.Web/wwwroot/css/about.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#profile-img:hover {
+    transform: translateY(-4px);
+    box-shadow: 0 4px 6px 2px rgba(0,0,0,0.5);
+}
+#sign-out {
+    display: none;
+}
+#callback-message {
+    display: none;
+}
```
Comment (itsmistad) [inline src/DnDTracker.Web/wwwroot/css/about.css]: None of these id selectors are used in the about page and are a duplicate of login.css. This file should be removed.

File: src/DnDTracker.Web/Views/About/Index.cshtml [hunk 2]
```diff
@@ -2,9 +2,41 @@
     ViewData["Title"] = "About";
 }
 
-<div id="about-page" class="full-center">
-    <div class="text-center">
-        <p>Now hold on, this hasn't been written yet!</p>
-        <p>🤯</p>
+@section Head
+{ 
+    <link rel="stylesheet" href="~/css/about.css" />
```
Comment (itsmistad) [inline src/DnDTracker.Web/Views/About/Index.cshtml]: This block should be removed. `about.css` does not apply to this page.

File: src/DnDTracker.Web/Views/About/Index.cshtml [hunk 3]
```diff
@@ -2,9 +2,41 @@
...
+}
+
+<div id="about-page">
+    <div class="column-container">
+        <div class="column full-center">
+            <div class="banner">
+                <div class="banner-content full-center">
+                    <p>
+                        <strong>DnDTracker</strong> was made by these four computer science students going to the University of North Texas:
```
Comment (itsmistad) [inline src/DnDTracker.Web/Views/About/Index.cshtml:11]: I think "was `developed` by" and "these four `University of North Texas students`:" flows a little better

**Review by nathanielduncan — DISMISSED** [review]
Looks good to me

**Review by itsmistad — CHANGES_REQUESTED** [review]

**Review by nathanielduncan — DISMISSED** [review]
Much different, unbelievable progress. Steller wording.

File: src/DnDTracker.Web/Views/About/Index.cshtml (hunk 2 above)
Comment (TylerCarey98) [reply src/DnDTracker.Web/Views/About/Index.cshtml]: How so?

**Review by itsmistad — APPROVED** [review]

**Pull request merged**